reseq_ros2/config/realsense_rgb_motion.yaml

```diff
@@ -8,4 +8,6 @@ realsense:
       enable_gyro: true
       enable_accel: true
 
-      rgb_camera.profile: 960x540x30
\ No newline at end of file
+      rgb_camera.profile: 960x540x30
+
+      wait_for_device_timeout: 20.0
```

Comment:
I believe that 10s are enough but it is not a problem

Reply:
Done